components/products/Product.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+import { Product } from '@prisma/client';
+
+defineProps<{
+  product: Product;
+}>()
+
+</script>
+
+<template>
+  <div class="product">
+
+    <img v-if="product.image" :src="product.image" alt="Product Image" class="image" />
+    <div v-else class="image-placeholder"></div>
+
+    <div class="product-info">
+      <h1>{{ product.title }}</h1>
+      <p class="price">${{ product.price }}</p>
+      <p class="description">{{ product.description }}</p>
+    </div>
+    
+  </div>
+</template>
+
+<style scoped>
+.product {
+  display: flex;
+  padding-bottom: 10px;
+  width: 100%;
+}
+
+.image {
+  margin-right: 20px;
+  width: 50%;
+}
+
+.image-placeholder {
```

Comment:
This class was meant to create a space for a missing image, when there is image missing in displayed product. To keep the structure symetric. It is definitely not ideal. It would be probably better to also include text saying " no image avaiable ", to explain the unused space to user.